.gitignore

```diff
@@ -28,3 +28,4 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+test/**
```

Comment:
Don't place the test directory into gitignore file. In future you will be in need of testing of your app. 
For now, 
1. you can ignore generating test files by learning "How you can skip generating test/spec files in rails" 
2. or you can delete them manually.